ubuntu/files/sources.list

```diff
@@ -0,0 +1,54 @@
+## Note that this sources.list file is picked from saltfs repository
```

Comment:
Note the original source instead (I'm assuming an Ubuntu VM).
It's also probably a good idea to add a note/warning that this file is managed by Salt.